src/main/java/org/fundacionjala/virtualassistant/player/spotify/client/SpotifyClient.java

```diff
@@ -0,0 +1,364 @@
+package org.fundacionjala.virtualassistant.player.spotify.client;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.servlet.view.RedirectView;
+
+import java.io.IOException;
+
+@Component
+public class SpotifyClient implements MusicClient {
+
+    @Value("${spotify.client.id}")
+    private String clientId;
+
+    @Value("${spotify.client.secret}")
+    private String clientSecret;
+
+    @Value("${spotify.redirect.uri}")
+    private String redirectUri;
+
+    private String accessToken;
+
+    @Override
+    public boolean playSongOnDevice(String trackUri) {
+        try {
+            HttpHeaders headers = new HttpHeaders();
+            headers.set("Authorization", "Bearer " + accessToken);
```

Comment:
dont use hardcoded strings here define a String final variable `final String AUTHORIZATION = "Authorization"`, same for BEARER